appveyor.yml

```diff
@@ -0,0 +1,43 @@
+version: 1.1.x.{build}
+skip_commits:
+  message: /(^\(?doc\)?.*|.*[A|a]cceptance [T|t]est.*)/
+clone_depth: 10
+init:
+- SET
+- 'mkdir C:\ProgramData\PuppetLabs\code && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\facter && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\hiera && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\puppet\var && exit 0'
+environment:
+  matrix:
+  - PUPPET_GEM_VERSION: 3.7.5
```

Comment:
All the way back to 3.3.0 you need https://github.com/puppetlabs/puppetlabs-registry/blob/master/metadata.json#L32. I'd do that, 3.8.1 and 4.1.0.